web/libs/editor/src/mixins/SelectedChoiceMixin.js

```diff
@@ -40,6 +39,12 @@ const SelectedChoiceMixin = types.model().views((self) => ({
       // as they are using alias lookups for choices. For now we will keep it as is since it works for all the
       // other cases currently.
       if (self.findLabel) {
+        // For Taxonomy, we need to check if any of the selected paths contain any of the choice values
+        if (self.type === "taxonomy") {
+          return selectedValues.some(path =>
+            Array.isArray(path) && choiceValue.some(value => path.includes(value))
```

Comment:
I think `path` will always be an array for taxonomies, so this check is excess, right? And if there are cases with non-array values, then we might need to handle them as well?

Reply:
Good improvement, thanks